src/utils/storage.py

```diff
@@ -11,6 +12,10 @@ def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
 
 
+class WhiteNoiseStaticFilesStorage(CompressedManifestStaticFilesStorage):
```

Comment:
maybe a comment about why we need this?

Reply:
just pulling since we're not even using this